.circleci/config.yml

```diff
@@ -3,44 +3,53 @@ version: 2.1
 jobs:
   test:
     docker:
-      - image: cimg/python:3.8
+      - image: circleci/node:latest
     steps:
       - checkout
-      - run: python --version
-      - run: pip install mkdocs==1.2.1 mkdocs-redirects mkdocs-meta-descriptions-plugin
-      - run: mkdocs build
+      - restore_cache:
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install Deps
+          command: yarn install
+      - save_cache:
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - node_modules
+      - run:
+          name: Yarn Build
+          command: yarn build
   deployStaging:
     docker:
-      - image: cimg/python:3.8
+      - image: circleci/node:latest
     steps:
       - checkout
-      - run: python --version
-      - run: pip install mkdocs==1.2.1 mkdocs-redirects mkdocs-meta-descriptions-plugin
-      - run: mkdocs build
+      - restore_cache:
+          key: dependency-cache-{{ checksum "yarn.lock" }}
+      - run:
+          name: Install aws cli
+          command: sudo apt-get -y -qq install awscli && aws --version
       - run:
-          name: install aws cli
-          command: |
-            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
-            unzip awscliv2.zip
-            sudo ./aws/install
-      - run: aws --version
-      - run: aws s3 sync site/ s3://staging-semgrep-dev/docs
+          name: Install Deps
+          command: yarn install
```

Comment:
Do you mean `yarn build`?

Reply:
```suggestion
          command: yarn build
```

Reply:
Yep it was missing the build and I think it needs both.